account_financial_report_webkit/report/common_partner_balance_reports.py

```diff
@@ -75,11 +75,16 @@ def _get_account_partners_details(self, account_by_ids, main_filter,
                             {'init_balance': initial_balances['init_balance']})
 
             # compute balance for the partner
+            partner_to_remove = []
             for partner_id, partner_details in details.iteritems():
                 details[partner_id]['balance'] = details[partner_id].\
                     get('init_balance', 0.0) + \
                     details[partner_id].get('debit', 0.0) - \
                     details[partner_id].get('credit', 0.0)
+                if not round(details[partner_id]['balance'], 4):
```

Comment:
I take exclude_null_balance into account now...
I am not sure how to use decimal_precision since I have no cursor in the method